src/labelle/lib/render_engines/margins.py

```diff
@@ -14,49 +13,64 @@
 )
 
 
+class LabelMarginsPx(NamedTuple):
+    horizontal: int
+    vertical: int
+
```

Comment:
According to Python docs, `NamedTuple` type hint is a "Typed version of [collections.namedtuple()](https://docs.python.org/3/library/collections.html#collections.namedtuple)". See https://docs.python.org/3/library/typing.html#typing.NamedTuple

Don't you prefer to use `collections.namedtuple`, instead of a regular Python class?

Reply:
Good question. Some benefits:

1. Types. With `collections.namedtuple` you just have names, without the types, so you wouldn't get type checking.
2. Syntax. This behaves and looks like a normal Python class, but the init and string representation are managed for you. This also makes it easy to switch out with similar data containers, like `pydantic.BaseModel` in case you need input validation.